LoggerApp.py

```diff
@@ -198,29 +124,14 @@ def calculate_id(token: str) -> str:
     return base64.b64encode(hashlib.sha256(token.encode('utf-8')).digest()).decode('utf-8')
 
 
-def wait_for_kafka(kafka_endpoint, timeout: float = 60) -> None:
```

Comment:
Why can we remove this waiting for kafka?

Reply:
This service can handle if kafka is down. In that case it will send an error to the requester.